tests/test_browser.py

```diff
@@ -3665,7 +3665,8 @@ def test_pthread_condition_variable(self):
   # Test that pthreads are able to do printf.
   @requires_threads
   def test_pthread_printf(self):
-    self.btest(path_from_root('tests', 'pthread', 'test_pthread_printf.cpp'), expected='0', args=['-s', 'TOTAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=1'])
+    for library_debug in [0, 1]:
```

Comment:
I find that the backtrace in the error reporting is nicer if you follow the pattern and avoid loops like this in the test code:

```
def run(debug):
   self.btest(path_from_root('tests', 'pthread', 'test_pthread_printf.cpp'), expected='0', args=['-s', 'TOTAL_MEMORY=64MB', '-O3', '-s', 'USE_PTHREADS=1', '-s', 'PTHREAD_POOL_SIZE=1', '-s', 'LIBRARY_DEBUG=%d' % debug])

run(debug=True)
run(debug=False)
```